MSAL/test/app/ios/MSALTestAppCacheViewController.m

```diff
@@ -53,12 +53,12 @@ @interface MSALTestAppCacheViewController () <UITableViewDataSource, UITableView
 @property (nonatomic) MSIDAccountCredentialCache *tokenCache;
 @property (nonatomic) MSIDDefaultTokenCacheAccessor *defaultAccessor;
 @property (nonatomic) MSIDLegacyTokenCacheAccessor *legacyAccessor;
+@property (strong, atomic) NSArray *accounts;
```

Comment:
nit: atomic is the default type, so you don't need to specify it that way.

Reply:
Also, I don't see a change of actually using property instead of instance variable in code :)